docs/antrea-network-policy.md

```diff
@@ -1102,6 +1107,35 @@ spec:
       - fqdn: "svcA.default.svc.cluster.local"
 ```
 
+## Node Selector
+
+NodeSelector selects certain Nodes which match the label selector. It adds Node IPs to egress rules in `to` field
+or ingress rules in `from` filed.
+The following rule applies to Pods with label `app=antrea-test-app` and will `Drop` egress traffic to
```

Comment:
If you want a new line, add an extra new line here.

Reply:
Done.